website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -53,7 +53,7 @@ All organizations in Terraform Cloud contain an “owners” team, which grants
 
 ~> **Note:** Teams not available to free-tier users on Terraform Cloud. Organizations at the free-level will only have an owners team.
 
-You can grant teams [read, plan, write, admin, or a customized set of permissions](/cloud-docs/users-teams-organizations/permissions#workspace-permissions) within a workspace. It’s important to note that, from a security perspective, the plan permission is equivalent to the write permission. The plan permission is provided to protect against accidental Terraform runs but is not intended to stop malicious actors from accessing sensitive data within a workspace.  Terraform `plan` and `apply` operations can execute arbitrary code within the ephemeral build environment. Both of these operations happen in the same security context with access to the full set of workspace variables, Terraform configuration, and Terraform state.
+All workspaces in an organization belong to a project. Teams may be granted read or admin permissions for the project, which grants the equivalent permission on all workspaces within the project. You can also grant teams [read, plan, write, admin, or a customized set of permissions](/cloud-docs/users-teams-organizations/permissions#workspace-permissions) for a specificworkspace. It’s important to note that, from a security perspective, the plan permission is equivalent to the write permission. The plan permission is provided to protect against accidental Terraform runs but is not intended to stop malicious actors from accessing sensitive data within a workspace. Terraform `plan` and `apply` operations can execute arbitrary code within the ephemeral build environment. Both of these operations happen in the same security context with access to the full set of workspace variables, Terraform configuration, and Terraform state.
```

Comment:
```suggestion
All workspaces in an organization belong to a project. Teams may be granted read or admin permissions for the project, which grants the equivalent permission on all workspaces within the project. You can also grant teams [read, plan, write, admin, or a customized set of permissions](/cloud-docs/users-teams-organizations/permissions#workspace-permissions) for a specific workspace. It’s important to note that, from a security perspective, the plan permission is equivalent to the write permission. The plan permission is provided to protect against accidental Terraform runs but is not intended to stop malicious actors from accessing sensitive data within a workspace. Terraform `plan` and `apply` operations can execute arbitrary code within the ephemeral build environment. Both of these operations happen in the same security context with access to the full set of workspace variables, Terraform configuration, and Terraform state.
```

Reply:
Changes made in this PR to be merged upon approval: https://github.com/hashicorp/terraform-docs-common/pull/245